interface/ceed-operator.c

```diff
@@ -983,6 +1194,183 @@ int CeedOperatorLinearAssembleAddPointBlockDiagonal(CeedOperator op,
   return 0;
 }
 
+/**
+  @brief Create a multigrid coarse operator and level transfer operators
+           for a CeedOperator with a Lagrange tensor basis for the active basis
+
+  @param[in] PMultFine    L-vector multiplicity in parallel gather/scatter
+  @param[in] rstrCoarse   Coarse grid restriction
+  @param[in] degreeCoarse Coarse grid basis polynomial order
```

Comment:
I am happy with the other changes - this should be the last open question, I believe.

For the MFEM integration, I'll want to use their 'transfer matrix', so I think that `CeedOperatorMultigridLevelCreateTensorH1` and `CeedOperatorMultigridLevelCreateH1` are what I will use for that.

That leaves replacing `CeedOperatorMultigridLevelCreateTensorH1Lagrange` with `CeedOperatorMultigridLevelCreate` where the transfer basis is automatically created out of the fine and coarse bases. It would be pretty flexible and would cover non-tensor and tensor bases. I think I should be able to use some of the same code in GetCollocatedGrad.

Reply:
Update - should be ready to go now